hasher-matcher-actioner/terraform/main.tf

```diff
@@ -218,6 +220,12 @@ module "authentication" {
   webapp_and_api_shared_user_pool_client_id = var.webapp_and_api_shared_user_pool_client_id
 }
 
+module "durable_fs" {
```

Comment:
I image this is not free, thoughts on an enable flag for tfvars?

Reply:
Setting up an EFS is not going to cost $$. IOPS are billed instead.

So, while I want to make this configurable, I'd rather do that at the individual feature level. 

eg. `tfvars.enable_live_content_clustering` determines whether we write and read. But the EFS exists either way.

I decided against having the EFS optional. It needs to be added to the hashing lambda, api_root and a few other lambdas. Making all of that configurable would have been an exercise in terraform dyanmics that's best avoided. 

LMK if you think otherwise.